File: Ejercicio2-EntradaSalida.cpp
/* 2. Escribe un programa que lea de la entrada estándar el precio de un producto y muestre en la salida estándar el precio del producto al aplicarle el IVA. */

#include<iostream>

using namespace std;

int main(){
    int precio;
    float iva;
    cout<<"Digite el precio del producto: "; cin>>precio;
    
    iva=precio+((precio*16)/100);
    cout<<"\nEl precio con iva es: $"<<iva<<endl;
    return 0;
}

File: Comparativa Python/Ejercicio2-EntradaSalida.py
"""2. Escribe un programa que lea de la entrada estándar el precio de un
producto y muestre en la salida estándar el precio del producto al aplicarle
el IVA."""

precio = float(input("Digite el precio del producto: "))

iva = precio*.16

print('El precio del producto con IVA es: $', precio+iva)
